ifrs17-template/Initialization/InitSystemorphUtils.ipynb

```diff
@@ -0,0 +1,170 @@
+                "\n                                                                                        (\"TransactionalData/NominalCashflows_ES_2020_12.csv\", ImportFormats.Cashflow), ",
+                "\n                                                                                        (\"TransactionalData/Actuals_ES_2020_12.csv\", ImportFormats.Actual)}}, ",
+                "\n                                                                    {\"FR\", new (string, string)[]{(\"TransactionalData/Openings_FR_2020_12.csv\", ImportFormats.Opening), ",
+                "\n                                                                                        (\"TransactionalData/NominalCashflows_FR_2020_12.csv\", ImportFormats.Cashflow), ",
+                "\n                                                                                        (\"TransactionalData/Actuals_FR_2020_12.csv\", ImportFormats.Actual)}}, ",
+                "\n                                                                    {\"CH\", new (string, string)[]{(\"TransactionalData/Openings_CH_2020_12.csv\", ImportFormats.Opening), ",
+                "\n                                                                                        (\"TransactionalData/NominalCashflows_CH_2020_12.csv\", ImportFormats.Cashflow),  ",
+                "\n                                                                                        (\"TransactionalData/Actuals_CH_2020_12.csv\", ImportFormats.Actual), ",
+                "\n                                                                                        (\"TransactionalData/SimpleValue_CH_2020_12.csv\", ImportFormats.SimpleValue), ",
+                "\n                                                                                        (\"TransactionalData/NominalCashflows_CH_2020_12_MTUP10pct.csv\", ImportFormats.Cashflow), ",
+                "\n                                                                                        (\"TransactionalData/NominalCashflows_CH_2021_3.csv\", ImportFormats.Cashflow), ",
+                "\n                                                                                        (\"TransactionalData/Actuals_CH_2021_3.csv\", ImportFormats.Actual)}}};"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Import Tasks"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public async Task InitializeDataNodesAsync(IDataSourceVariable dataSource, string pathToImport, string[] nodes = null){",
+                "\n    nodes = nodes ?? new string[]{\"ES\", \"FR\", \"CH\"};",
```

Comment:
Take the keys of the dict instead of hardcoding the reportingNodes

Reply:
Done.